packages/web-components/src/components/rux-radio-group/test/radio-group.spec.ts

```diff
@@ -191,28 +191,166 @@ test.describe('Radio-group-with-form', () => {
 test.describe('Radio-group', () => {
     test.beforeEach(async ({ page }) => {
         const template = `
-        <rux-radio-group label="hello">
-            <rux-radio label="first" value="first"></rux-radio>
-            <rux-radio label="second" value="second"></rux-radio>
-        </rux-radio-group>
-        <rux-radio-group><div slot="label">hello</div></rux-radio-group>
+            <rux-radio-group name="radios" id="no-value">
+              <rux-radio>One</rux-radio>
+              <rux-radio>Two</rux-radio>
+              <rux-radio>Three</rux-radio>
+            </rux-radio-group>
+
+            <h2 id="focus-title">Focus Testing</h2>
+            <rux-radio-group name="radios-check-one" id="focus-check-one">
+              <rux-radio value="one">One</rux-radio>
+              <rux-radio value="two">Two</rux-radio>
+              <rux-radio value="three" disabled id="disabled-radio">Three</rux-radio>
+            </rux-radio-group>
+            <rux-radio-group name="radios-check-two" id="focus-check-two">
+              <rux-radio value="one" id="group2-radio1">One</rux-radio>
+              <rux-radio value="two">Two</rux-radio>
+              <rux-radio value="three">Three</rux-radio>
+            </rux-radio-group>
+            <rux-radio-group id="first-test" label="hello">
+              <rux-radio value="first"></rux-radio>
+              <rux-radio value="second"></rux-radio>
+            </rux-radio-group>
+            <rux-radio-group><div slot="label">hello</div></rux-radio-group>
         `
         await page.setContent(template)
     })
     test('renders label prop and slot', async ({ page }) => {
         //Arrange
-        const ruxRadioProp = await page.locator('rux-radio-group').first()
+        const ruxRadioProp = page.locator('rux-radio-group').first()
         const labelProp = ruxRadioProp.locator('.rux-label')
-        const ruxRadioSlot = await page.locator('rux-radio-group').nth(1)
+        const ruxRadioSlot = page.locator('rux-radio-group').nth(1)
```

Comment:
this test is failing now. think it might have to do with the changes to the template content. rather than try and use the same template for every test, its okay to split it up so each test has its own template setup, even if it means duplicating some.